main.cpp

```diff
@@ -280,6 +309,31 @@ void School::addCourse(std::string courseName, int gradeLevel)
         numCourses -= 1;
 }
 
+School::Teacher School::Teacher::returnTeacher(int threshold, int startingValue)
+{
+    Teacher t1;
+    while (startingValue < threshold)
+    {
+        t1.numWorkingYears += 10;
+        if (startingValue + threshold < t1.numWorkingYears)
+            return t1;
+    }
+    return t1;
+}
+
+School School::returnSchool(int threshold, int startingValue)
+{
+    School s1;
+    for (int i = startingValue; i < threshold; startingValue+=2)
+    {
+        s1.numTeachers += 1;
+        s1.numStudents += 3;
+        if (startingValue <= s1.numStudents)
+            return s1;
+    }
+    return s1;
```

Comment:
You have 2 identical return statements.
Don't Repeat Yourself. 
Instead, `break` out of the `for()` loop. 
```
if( whatever )
    break;
```
- [ ] revise